### ISSUE TITLE
How Frequently to Update Prophet Model

### ISSUE THREAD

**Issue by chessplayer2** [facebook/prophet#838]
Hi there. How frequently should one update their prophet model?

I am currently running a model that forecasts data on a monthly level. I tested my prophet on a training set of 4 years worth of data (48 observations). On my test validation of data of 6 months, the MAPE is well within 1-2%. However, I am curious to know how frequently one should go back to their prophet and update the model (ie n.changepoints, changepoint.range, etc.).

Also, should I be concerned if my prophet model has a MAPE of 15-20% on the training data set but 1-2% on the test data set?

**Comment by bletham** [comment]
My personal experience is that hyperparameters like prior scales changepoints are pretty stable over time and I wouldn't think you'd need to re-tune them very often unless you do find there is a significant change in the nature of the time series.

For the fitting, that's a bit surprising since usually training error is less than test error. How are you computing test error? I'm wondering if perhaps there are a few outliers or bad points in the training set that are not present in the test set that are causing the issue. I think if you compute test error, and then include those data in the training set and compute error on those same points then it should be less?
